operatorcourier/validate.py

```diff
@@ -27,13 +51,15 @@ def validate(self, bundle):
         """
         logger.info("Validating bundle.")
 
-        return self._bundle_validation(bundle)
+        self._bundle_validation(bundle)
+
+        return self._bundle_validation(bundle), self.validationJSON
```

Comment:
Why is `self._bundle_validation(bundle)` called twice?

Reply:
This is why we see doubled error output in example https://github.com/operator-framework/operator-courier/pull/27#issuecomment-471027625

Reply:
Thanks for catching this